Days-Calculator.html

```diff
@@ -103,6 +105,7 @@
             <button type="button" onclick="DaysCalculator()">Submit</button><br/><br/><br/>
             <span id="resultAge"></span>
         </form>
+        <a href="./index.html"><i id="home" class="fa fa-home" aria-hidden="true"></i></a>
```

Comment:
the current codebase in your recent version does not contain this line, probably some other PR overwrote these lines.